projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts

```diff
@@ -92,8 +92,6 @@ describe('CaseFileViewFolderComponent', () => {
     await fixture.whenStable();
     fixture.detectChanges();
     expect(component.filter).toHaveBeenCalled();
-
-    expect(component.documentTreeData).toEqual(treeData);
```

Comment:
I understand why you've removed this check, but you should replace it with something. For example you could check that the first item is the appropriately sorted first item